src/pages/Customer/LoginPage/LoginPage.js

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from "react";
+import styles from "./LoginPage.module.scss";
+import bgSignin from "assets/images/bgSignin.png";
+import Button from "components/Button/Button";
+import InputControl from "components/InputControl/InputControl";
+import { useNavigate } from "react-router-dom";
+import VerifyOtp from "components/verifyOtp/VerifyOtp";
+import { checkMobile, login } from "api/user/login";
+import { validateMobile } from "utils/util";
```

Comment:
for all the imports in the app we are following a structure like library imports -> component imports -> asset imports -> css imports